### DISCUSSION TITLE
selenium grid

### DISCUSSION THREAD

**Discussion by Stevenphil** [seleniumbase/SeleniumBase#1163] [General]
First of all, amazing work!
When I execute test cases concurrently on selenium grid, the chrome page always crashes.
eg: _pytest   --server="9.134.125.232" --port=5555 -n=5_

**Comment by mdmintz** [comment]
Hi @Stevenphil, I would need more info to help debug. Do you get the same issues on a locally hosted Selenium Grid? Is there a stack trace with more info? Does the crash happen before the test starts, or in the middle? Is the Grid configured properly? Are the Grid Nodes on a headless machine (and therefore you may need to add ``--headless`` as an argument)?
